config/config_test.go

```diff
@@ -496,8 +495,6 @@ func TestConflictInstanceConfig(t *testing.T) {
 	var expectedNewName string
 	conf := new(Config)
 	configFile := "config.toml"
-	_, localFile, _, _ := runtime.Caller(0)
```

Comment:
Why make these changes?

Reply:
It is useless code. but i think I can create a pr to merge it.